src/mplfinance/plotting.py

```diff
@@ -76,7 +79,7 @@ def _valid_plot_kwargs():
         'mav'         : { 'Default'     : None,
                           'Validator'   : _mav_validator },
         
-        'renko_params': { 'Default'     : dict(),
+        'pmove_params': { 'Default'     : dict(),
```

Comment:
I just now realized we this table should (at least for some period of time, maybe a few months) have BOTH `pmove_params` and `renko_params`, and the Validator for `renko_params` should give a warning message telling the caller to use `pmove_params` instead.  

There may be users out there who have written code for renko plots, and when they upgrade to the next version of mplfinance, we don't want their code to stop working, *at least not without telling them explicitly how to fix it* (so they don't have to debug to figure out why renko_params is no longer a valid kwarg).  

Maybe something along the lines of:

    'kwarg `renko_params` has been renamed to `pmove_params` 
    in order to be generally used for all 'price-movement' based plot types.  
    Please use `pmove_params` instead.'

This is similar to what I did when deprecating `no_xgaps` (see [here](https://github.com/matplotlib/mplfinance/blob/master/src/mplfinance/plotting.py#L88-L91) and [here](https://github.com/matplotlib/mplfinance/blob/master/src/mplfinance/plotting.py#L43-L51) **however** `renko_params` is ***not*** strictly being deprecated, but is just being renamed.  Or *should* we say that it is deprecated and replaced with `pmove_params` ... I'm not sure what to say.  I don't see a "renamed" warning listed among the [Python warning classes](https://docs.python.org/3/library/warnings.html).  **Perhaps we say it was renamed, but still set the warning category to `DeprecationWarning`** because we don't want them to use it anymore.   ***Or do you*** want to let them use either `renko` or `pmove` params??  What do you think?